WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/filter/ActivityLogTypeFilterFragment.kt

```diff
@@ -48,7 +45,7 @@ class ActivityLogTypeFilterFragment : DialogFragment() {
         super.onViewCreated(view, savedInstanceState)
         initToolbar(view)
         initRecyclerView()
-        viewModel.start(remoteSiteId = RemoteId(requireNotNull(arguments).getLong(WordPress.REMOTE_SITE_ID)))
+        initViewModel()
     }
 
     private fun initToolbar(view: View) {
```

Comment:
This is not related to changes in this PR and should've been caught earlier, can we remove unused view param since we've replaced findByViewId with Kotlin synthetics.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/13469/commits/15ef9fdbd4a974ff07f0c4e755d89c63115b4179